client/client.go

```diff
@@ -505,12 +508,12 @@ func tsLessEqual(physical, logical, thatPhysical, thatLogical int64) bool {
 	return physical < thatPhysical
 }
 
-func (c *client) finishTSORequest(requests []*tsoRequest, physical, firstLogical int64, err error) {
+func (c *client) finishTSORequest(requests []*tsoRequest, physical, firstLogical int64, suffixBits uint32, err error) {
 	for i := 0; i < len(requests); i++ {
 		if span := opentracing.SpanFromContext(requests[i].requestCtx); span != nil {
 			span.Finish()
 		}
-		requests[i].physical, requests[i].logical = physical, firstLogical+int64(i)
+		requests[i].physical, requests[i].logical = physical, firstLogical+int64(i<<suffixBits)
```

Comment:
It confused me that you say `firstLogical` is the highest ts on L473, while each request's `logical` is equal or bigger than `firstLogical`.

Reply:
I revised the wording of the comment. PTAL again!